query-engine/connector-test-kit-rs/query-tests-setup/src/datamodel_rendering/mod.rs

```diff
@@ -37,6 +37,7 @@ pub fn render_test_datamodel(
     excluded_features: &[&str],
     relation_mode_override: Option<String>,
     db_schemas: &[&str],
+    isolation_level: Option<&'static str>,
```

Comment:
Does the isolation need to be in the connection string? Could it be set after the connection is established, in the connection? Ultimately, I think we want to stop constructing connection strings in query engine test setup code

Reply:
Yeah, that would be really nice. There is a mechanism in quaint that sets the isolation level, if it's easy enough I'd like to use that. If not, we'll go with this.